### PR TITLE
Update soroban-client to 0.2.0

### PR DESCRIPTION
**Pull request by paulbellamy**
- [x] Also need to update @soroban-react with the new version once https://github.com/esteblock/soroban-react/pull/1 is merged/released.

This might be broken until new freighter is released.

### CODE REVIEW

**Review by leighmcculloch — APPROVED** [review]

File: convert.ts [hunk 1]
```diff
@@ -3,44 +3,96 @@ import * as SorobanClient from 'soroban-client';
...
+  }
+
+  // left-pad with zeros up to 4 bytes
+  let padded = Buffer.alloc(4);
+  buf.copy(padded, padded.length-buf.length);
+
+  if (value.isNegative()) {
+    // Set the top bit
+    padded[0] |= 0x80;
+  }
```
Comment (leighmcculloch) [inline convert.ts:87]: Why is this needed?

Comment (paulbellamy) [reply convert.ts:87]: My understanding of the Int128Parts was that if it is negative the top bit is set?

Comment (leighmcculloch) [reply convert.ts:87]: Ah right, got it.
